FirstProject/TuttinFruttiProgram/TuttiFrutti.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TuttiFruttiProgram
+{
+    class TuttiFrutti
+    {
+        public void outputValues()
+        {
+            string value1 = "Tutti";
```

Comment:
You can declare this values not in method (in the class).
You will have ability to use them in other methods.